reporters/GlobalSummaryReporter.js

```diff
@@ -377,47 +396,43 @@ class GlobalSummaryReporter {
   async _filterDisputeData(periods, disputeEvents, liquidationEvents) {
     let allUniqueDisputes = {};
     let periodUniqueDisputes = {};
-    let allTokensDisputed = this.toBN("0");
+    let allTokensDisputed = 0;
     const periodTokensDisputed = {};
-    let allCollateralDisputed = this.toBN("0");
+    let allCollateralDisputed = 0;
     const periodCollateralDisputed = {};
     const allResolvedDisputes = {};
 
     for (let event of disputeEvents) {
       // Fetch disputed collateral & token amounts from corresponding liquidation event that with same ID and sponsor.
       const liquidationData = liquidationEvents.filter(
-        e => e.liquidationId === event.liquidationId && e.sponsor === event.sponsor
+        e => e.liquidationId === event.liquidationId && e.sponsor.id === event.sponsor.toLowerCase()
```

Comment:
Historically we've prefered to use: `web3.utils.toChecksumAddress`

Reply:
Well the subgraph always returns addresses that are all lower cased (i.e. NOT checksummed). This is equivalent to comparing `toChecksumAddress(e.sponsor.id) === event.sponsor`. WDYT?

Reply:
I see. Seeing you lose the checksum because all lower case in the return type the implementation is good to go then 👍